packages/automation/src/mapping.ts

```diff
@@ -33,6 +33,10 @@ export const commandAddressMapping: Record<
       '0x02b7391cdd0c8a75ecfc278d387e3dcc3d796340': {
         type: CommandContractType.AutoTakeProfitCommand,
       },
+      '0xc3c42b67e4cbd6226b743352144d4b285c851f6e': {
+        type: CommandContractType.AutoTakeProfitCommand,
+        overwrite: ['uint256', 'uint16', 'uint256', 'uint32'],
```

Comment:
this part is not needed they all have same interface

Reply:
gotcha!